### PR TITLE
Add automatic owner detection

### PR DESCRIPTION
**Pull request by S4mpsa**

### CODE REVIEW

File: hud/powerdisplay.lua [hunk 1]
```diff
@@ -146,17 +169,24 @@ function powerDisplay.widget(glasses, data)
...
+                for j = 1, #hudObjects[i].static do
+                    hudObjects[i].glasses.removeObject(hudObjects[i].static[j].getID())
+                end
+                hudObjects[i].glasses.removeObject(hudObjects[i].dynamic.energyBar.getID())
+                hudObjects[i].glasses.removeObject(hudObjects[i].dynamic.currentEU.getID())
+                hudObjects[i].glasses.removeObject(hudObjects[i].dynamic.maxEU.getID())
+                hudObjects[i].glasses.removeObject(hudObjects[i].dynamic.percentage.getID())
+                hudObjects[i].glasses.removeObject(hudObjects[i].dynamic.filltime.getID())
+                hudObjects[i].glasses.removeObject(hudObjects[i].dynamic.fillrate.getID())
+                hudObjects[i].glasses.removeObject(hudObjects[i].dynamic.state.getID())
```
Comment (gordominossi) [inline hud/powerdisplay.lua:187]: Could you iterate through the `dynamic` table?

File: hud/toolbar.lua [hunk 2]
```diff
@@ -67,14 +93,14 @@ function toolbar.widget(glasses)
...
+            hudObjects[i].dynamic.topstrip = ar.rectangle(hudObjects[i].glasses, {x+4, y+1}, w-4, 1, primaryColor) -- Top Strip
+            hudObjects[i].dynamic.diagonal = ar.quad(hudObjects[i].glasses, {x+w, y+1}, {x+w, y+2}, {x+w+rightTriangleSide-1, y+1+rightTriangleSide},  {x+w+rightTriangleSide-1, y+rightTriangleSide}, primaryColor) --Diagonal Strip
+            hudObjects[i].dynamic.rightstrip = ar.rectangle(hudObjects[i].glasses, {x+w+rightTriangleSide-2, y+rightTriangleSide}, 1, h-rightTriangleSide, primaryColor) --Right Strip
+            hudObjects[i].dynamic.clockEdge =  ar.rectangle(hudObjects[i].glasses, {x+w+4, y+10}, clockWidth, clockheight, primaryColor) --Clock Edge
             table.insert(hudObjects[i].static, ar.rectangle(hudObjects[i].glasses, {x+w+5, y+11}, clockWidth-2, clockheight-2, borderColor)) --Clock Panel
             hudObjects[i].dynamic.clock = ar.text(hudObjects[i].glasses, "", {x+w+7, y+13}, accentColor, 0.65)
             hudObjects[i].dynamic.realtime = ar.text(hudObjects[i].glasses, "", {x+w+3, y+30}, primaryColor, 0.8)
```
Comment (gordominossi) [inline hud/toolbar.lua:106]: Nice variable names and comments :+1:

File: hud/powerdisplay.lua (hunk 1 above)
Comment (S4mpsa) [reply hud/powerdisplay.lua:187]: Maybe, but I don't know how to do that off the top of my head. Maybe with pairs()?

File: lib/graphics/renderer.lua [hunk 3]
```diff
@@ -34,6 +34,34 @@ function renderer.leaveFocus()
     focused = false
 end
 
+function event.onError(message)
+    print(message)
+    graphics.text(1, graphics.context().heigth, message)
+end
+
+function renderer.multicast()
```
[the remark this answers is not in the document]
Comment (S4mpsa) [reply lib/graphics/renderer.lua:42]: Yes, mostly it's there so you can use a remote terminal at the same time as the main screen.

File: hud/powerdisplay.lua (hunk 1 above)
Comment (gordominossi) [reply hud/powerdisplay.lua:187]: Yes, probably

File: lib/graphics/renderer.lua (hunk 3 above)
[the remark this answers is not in the document]
Comment (gordominossi) [reply lib/graphics/renderer.lua:42]: Got it

**Pull request merged**